core/src/models/Export.ts

```diff
@@ -291,6 +293,38 @@ export class Export extends CommonModel<Export> {
     }
   }
 
+  @AfterUpdate
+  static async logExport(instance: Export) {
+    if (!process.env.GROUPAROO_EXPORT_LOG) return;
+
+    if (
+      instance.changed("state") &&
+      ["canceled", "failed", "complete"].includes(instance.state)
+    ) {
+      const exportData = {
+        id: instance.id,
+        state: instance.state,
+        recordId: instance.recordId,
+        modelId: (await instance.$get("record")).modelId,
+        destinationId: instance.destinationId,
+        newRecordProperties: instance.newRecordProperties,
+        newGroups: instance.newGroups,
+        toDelete: instance.toDelete,
+        errorMessage: instance.errorMessage,
+        errorLevel: instance.errorLevel,
+        timestamp: new Date().toISOString(),
+      };
+
+      const message = JSON.stringify(exportData);
+
+      if (process.env.GROUPAROO_EXPORT_LOG === "stdout") {
+        log(`[ export ] ${message}`);
+      } else {
+        await appendFile(process.env.GROUPAROO_EXPORT_LOG, `${message}\n`);
```

Comment:
Presumably if someone gave `GROUPAROO_EXPORT_LOG=log.log` they would want that log file to end up in `process.cwd()/log.log` not `/log.log`.  We do this relative path checking elsewhere in the project, like when loading configs or setting the sqlite database path.  

Check out https://github.com/grouparoo/grouparoo/blob/cbba7e7c96736f074dcd9b09310279dbab80ade6/core/src/config/sequelize.ts#L91-L94